README.md

```diff
@@ -154,13 +189,13 @@ Removes the ecrypted value in the platform secure storage.
 biometryService.Remove("KeyName");
 ```
 
-#### Android
+On Android, remove the encrypted data from the shared preferences.
```

Comment:
These sentences sound like you're giving an order to the reader.

It should be something like
> On Android **this method removes** the encrypted data....

Reply:
This same comment applies to the other methods.